docs/advisories/meltdown-kernel-update.md

```diff
@@ -43,7 +38,8 @@ dmesg -H | grep 'page tables isolation'
 
 ## Impacted Maintained Component(s)
 
-- kops maintained AMI
+* Patches were released for the linux kernel 2018-01-05.  All images prior to this date likely need updates.
```

Comment:
This is good information, but it does not match the heading.

Reply:
Not sure what you mean